### PR TITLE
RGB composite operator

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[RGB Layer im UI](https://trello.com/c/h0b5eJld/1348-rgb-layer-im-ui)

File: operators/src/processing/rgb.rs [hunk 1]
```diff
@@ -0,0 +1,615 @@
...
+impl OperatorData for RgbSources {
+    fn data_names_collect(&self, data_names: &mut Vec<NamedData>) {
+        for source in self.iter() {
+            source.data_names_collect(data_names);
+        }
+    }
+}
+
+impl RgbSources {
+    pub fn new_a(
```
Comment (jdroenner) [inline operators/src/processing/rgb.rs:120]: why is it called "new_a"? Is there a "b"?

File: operators/src/processing/rgb.rs [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+}
+
+fn compute_tile(
+    [red, green, blue]: [RasterTile2D<f64>; 3],
+    params: &RgbParams,
+) -> RasterTile2D<u32> {
+    fn fit_to_interval_0_255(value: f64, min: f64, max: f64, scale: f64) -> u32 {
+        let mut result = value - min; // shift towards zero
+        result /= max - min; // normalize to [0, 1]
+        result *= scale; // after scaling, value stays in [0, 1]
```
Comment (jdroenner) [inline operators/src/processing/rgb.rs:316]: add: "because scale is also  [0, 1]"

File: operators/src/processing/rgb.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply operators/src/processing/rgb.rs]: [d397cc5](https://github.com/geo-engine/geoengine/pull/807/commits/d397cc52ce31074c401214b88537b8fa5a42fb26)

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
i think you have to solve the problem with the cache hint missing

**Review by jdroenner — APPROVED** [review]

**Pull request merged**